metaflow/flowspec.py

```diff
@@ -419,6 +419,8 @@ def merge_artifacts(
             This exception is thrown in case an artifact specified in `include` cannot
             be found.
         """
+        INTERNAL_ARTIFACTS_SET = set(["_foreach_values"])
```

Comment:
nit: move this out of the function so it doesn't get recreated every time.

Reply:
updated